HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -24,6 +24,13 @@ def esproducers_by_type(process, *types):
 #                     pset.minGoodStripCharge = cms.PSet(refToPSet_ = cms.string('HLTSiStripClusterChargeCutNone'))
 #     return process
 
+def customiseFor14282(process):
```

Comment:
No problem: now that #14201 is in 80X, you can immediately start producing the Empty source as here and everyone will ignore it until this PR is also back-ported to 80x.